bin/configs/scala-http4s-server.yaml

```diff
@@ -0,0 +1,6 @@
+generatorName: scala-http4s-server
+outputDir: samples/server/petstore/scala-http4s-server
```

Comment:
can you please commit the new files for scala-http4s-server petstore as well?

then CI will run the tests on the new sample.

Reply:
Done